apps/system/js/statusbar.js

```diff
@@ -431,6 +440,12 @@
         case 'cardviewshown':
         case 'cardviewclosed':
           return 'cardview';
+        case 'appwillopen':
+        case 'stackchanged':
+          return 'appopening';
+        case 'appwillclose':
+        case 'stackchanged':
```

Comment:
hum... how is this supposed to work? The group will always be appopening for stackchanged...